java/src/main/java/org/synbiohub/PrepareSubmissionJob.java

```diff
@@ -482,8 +484,14 @@ public void visit(Identified identified, TopLevel topLevel) {
 						SBOLDocument tlDoc;
 						try {
 							tlDoc = sbh.getSBOL(URI.create(topLevelUri));
-						} catch (SynBioHubException e) {
-							tlDoc = null;
+						} catch (SynBioHubException e1) {
```

Comment:
Won't this still fail in the event of a NotFoundException?

Reply:
I changed libSBOLj to return null on NotFoundException.